Upgraded the baseweather widget

kabturek opened the pull request:
* added the ability to change the sensors used
* added dynamic units for more sensors
* added option for showing forecast (show_forecast: 1)
* added icons for conditions (prefer_icons: 1)
* added the option to show Rain/Snow depending on precip_type sensor (and change icons)
* wind icon rotates acording to wind bearing

There are some css changes (used 4 spaces instead of tabs). 
Had to remove the indentions from the html file to get rid of those ugly spaces between display-block elements.

homeassistant commented:
Hello @kabturek,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`7bcd615f1e4aeb5e928ffa390b287df9ba296c19`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;appdaemon&#x2F;pull&#x2F;270&#x2F;commits&#x2F;7bcd615f1e4aeb5e928ffa390b287df9ba296c19) - This commit has something that looks like an email address (marcin.domanski@kontomatik.com). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

homeassistant commented:
Hello @kabturek,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`c99e9691020edd4eedbd5fe249b6ae0a2c2902c6`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;appdaemon&#x2F;pull&#x2F;270&#x2F;commits&#x2F;c99e9691020edd4eedbd5fe249b6ae0a2c2902c6) - This commit has something that looks like an email address (marcin.domanski@kontomatik.com). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

homeassistant commented:
Hi @kabturek, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;appdaemon#270).

Once you do that we will be able to review and accept this pull request.

Thanks!

acockburn commented:
Hi there - I can't merge this until you fix the CLA error.

kabturek commented:
hey @acockburn - i've squashed the 3 commits to 1 and corrected the author email (was using a work machine) but it doesnt look like the bot removed the cla-error

kabturek commented:
i've asked on discord (#dev) and the bot wont automatically remove it. the cla-bot check passed so i would guess that everything is ok now (should be - i did everything from the bots message)
